src/sentry/api/serializers/models/incident.py

```diff
@@ -90,8 +90,14 @@ def serialize(self, obj, attrs, user):
 
     def _build_discover_query(self, incident):
         query = incident.alert_rule.snuba_query.query
-        if QueryDatasets(incident.alert_rule.snuba_query.dataset) == QueryDatasets.EVENTS:
+        dataset = QueryDatasets(incident.alert_rule.snuba_query.dataset)
+
```

Comment:
Define `condition = None` here, otherwise things would break if we added a third dataset, not that it's likely any time soon.

Reply:
👍 
\